conf/schema.yml

```diff
@@ -0,0 +1,196 @@
+                type: map
+                mapping:
+                  max_params:
+                    type: number
+      LongYieldList:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_params:
+            type: number
+      ManualDispatch:
+        type: map
+        mapping:
+          <<: *detector_base
+      MissingSafeMethod:
+        type: map
+        mapping:
+          <<: *detector_base
+      ModuleInitialize:
+        type: map
+        mapping:
+          <<: *detector_base
+      NestedIterators:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_allowed_nesting:
+            type: number
+          ignore_iterators:
+            type: seq
+            sequence:
+              - type: str
+      NilCheck:
+        type: map
+        mapping:
+          <<: *detector_base
+      RepeatedConditional:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_ifs:
+            type: number
+      SubclassedFromCoreClass:
+        type: map
+        mapping:
+          <<: *detector_base
+      Syntax:
+        type: map
+        mapping:
+          <<: *detector_base
+      TooManyConstants:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_constants:
+            type: number
+      TooManyInstanceVariables:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_instance_variables:
+            type: number
+      TooManyMethods:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_methods:
+            type: number
+      TooManyStatements:
+        type: map
+        mapping:
+          <<: *detector_base
+          max_statements:
+            type: number
+      UncommunicativeMethodName:
+        type: map
+        mapping:
+          <<: *detector_base
+          reject: &reject_settings
+            type: seq
+            sequence:
+              - type: str
+          accept: &accept_settings
+            type: seq
+            sequence:
+              - type: str
+      UncommunicativeModuleName:
+        type: map
+        mapping:
+          <<: *detector_base
+          reject: *reject_settings
+          accept: *accept_settings
+      UncommunicativeParameterName:
+        type: map
+        mapping:
+          <<: *detector_base
+          reject: *reject_settings
+          accept: *accept_settings
+      UncommunicativeVariableName:
+        type: map
+        mapping:
+          <<: *detector_base
+          reject: *reject_settings
+          accept: *accept_settings
+      UnusedParameters:
+        type: map
+        mapping:
+          <<: *detector_base
+      UnusedPrivateMethod:
+        type: map
+        mapping:
+          <<: *detector_base
+      UtilityFunction:
+        type: map
+        mapping:
+          <<: *detector_base
+          public_methods_only:
+            type: bool
+
+  "directories":
```

Comment:
Actually kwalify seems to be pretty awesome. Those half a dozen lines below validate all directory directives ❤️